app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -395,30 +402,27 @@ export default function Navigation() {
         </div>
 
         <Hidden smDown>
-          <div ref={menuRef} style={{ display: "flex", flexDirection: "row" }}>
-            <div
-              style={{ marginRight: 16, display: "flex", alignItems: "center" }}
-            >
-              <ReportButton />
-            </div>
-            <div
+          <div className={classes.menuContainer} ref={menuRef}>
+            <ReportButton />
+            <Button
               className={classes.menuBtn}
-              onClick={menuOpen ? handleMenuClose : handleMenuOpen}
+              onClick={() =>
+                setMenuOpen((prevMenuOpen: boolean) => !prevMenuOpen)
```

Comment:
the boolean can be inferred (not a bigdeal)

Reply:
Yes, seems like the array is working